templates/azure/confidential-computing.html

```diff
@@ -0,0 +1,181 @@
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Get Ubuntu confidential VMs on the Microsoft Azure marketplace</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted" style="margin-top: 0.55rem;">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/bee89574-azure-logo-updated.png"
+                 alt="" />
+          </div>
+        </div>
+
+        <p>Ubuntu offers the following confidential VMs:</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item has-bullet">
+            <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview">Get Ubuntu 24.04 LTS confidential VMs</a>
+          </li>
+          <li class="p-list__item has-bullet">
+            <a href="https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR9_qJVF-zyZCqvwR_iaG2EFUME1BT1hJN0I1VjhGWE9QNUJGREQ5QUlMNS4u">Get Intel TDX in private preview with 22.04 LTS</a>
+          </li>
+          <li class="p-list__item has-bullet">
+            <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-jammy">Get AMD SEV SNP with Ubuntu 22.04 LTS on Azure</a>
+          </li>
+          <li class="p-list__item has-bullet">
+            <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-focal?tab=Overview">Get AMD SEV SNP with Ubuntu  20.04 LTS on Azure</a>
+          </li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <a href="/confidential-computing#get-in-touch">Don’t see what you’re looking for? Contact us to learn more&nbsp;&rsaquo;</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Use Ubuntu Pro to further harden your confidential VMs</h2>
+      </div>
+      <div class="col">
+        <p>
+          While confidential VMs can protect your workload from external threats, vulnerabilities from within their boundaries remain a concern. With Ubuntu Pro, you can keep your guest CVM software stack continuously patched and up-to-date for up to 12 years, covering 30,000 open source packages with a single subscription.
+        </p>
+        <hr class="p-rule--muted" />
+        <a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
+      </div>
+    </div>
+  </section>
+
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://ubuntu.com/confidential-computing">Learn about other confidential computing offerings from Canonical&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
```

Comment:
```suggestion
    <div class="u-fixed-width">
```
Design doesn't have shallow padding for the header row